src/main/java/org/example/App.java

```diff
@@ -32,10 +38,6 @@ public static void main( String[] args ) {
 
 
         List<Route> allRoutes = routeService.getRoutes();
-        logger.info("*** ROUTES IN DATABASE ***");
-        for (Route rou : allRoutes)
-            logger.info(rou);
-
 
         List<Point> allPoints = pointService.getPoints();
         System.out.println("\n✦✦✦ LIST OF CITIES ✦✦✦");
```

Comment:
make it System.out.println("\nWhere would you like to go?"); to separate from previous text